secure_message/resources/threads.py

```diff
@@ -113,16 +116,26 @@ def get():
             messages = add_users_and_business_details(messages)
         return jsonify({"messages": messages, "_links": links})
 
+    @staticmethod
+    def _validate_request(request_args, user):
+        if request_args.my_conversations and user.is_respondent:
+            logger.error('My conversations option not available to respondents')
```

Comment:
```suggestion
            logger.error('My conversations option not available to respondents', user_uuid=user.user_uuid)
```

Should log out the user so if we need to follow up (say, a user is trying to probe our system, causing lots of errors), then we know who's doing it

Reply:
Added